pkg/cidata/cidata.TEMPLATE.d/boot.sh

```diff
@@ -10,7 +10,9 @@ WARNING() {
 }
 
 # shellcheck disable=SC2163
-while read -r line; do export "$line"; done <"${LIMA_CIDATA_MNT}"/lima.env
+while read -r line; do [ -n "$line" ] && export "$line"; done <"${LIMA_CIDATA_MNT}"/lima.env
+# shellcheck disable=SC2163
+while read -r line; do [ -n "$line" ] && export "$line"; done <"${LIMA_CIDATA_MNT}"/param.env
```

Comment:
> This works on `mode: dependency` and `mode: system`, but not on `mode: boot` and `mode: user`.

Thanks for catching that! I forgot about `boot`, but thought it would work for `user`.

> That should be documented or implemented.

I've implemented it for all modes, and added a test in the `test-misc.yaml` template to verify that they all work.